### PR TITLE
Demonstrate that plain unittest does not support async tests

### PR DESCRIPTION
**Pull request by nicoddemus**
Related to pytest-dev/pytest-asyncio#180

### CODE REVIEW

File: testing/test_unittest.py [hunk 1]
```diff
@@ -1243,3 +1243,19 @@ def test_asynctest_support(testdir):
     testdir.copy_example("unittest/test_unittest_asynctest.py")
     reprec = testdir.inline_run()
     reprec.assertoutcome(failed=1, passed=2)
+
+
+def test_plain_unittest_does_not_support_async(testdir):
+    """
+    Async functions in plain unittest.TestCase subclasses are not supported without plugins.
```
Comment (bluetech) [inline testing/test_unittest.py:1250]:
```suggestion
    """Async functions in plain unittest.TestCase subclasses are not supported without plugins.
```

:wink:

Comment (nicoddemus) [reply testing/test_unittest.py]: Oops thanks!

**Comment by bluetech** [comment]
@nicoddemus looks like the pypy3 failure here was legitimate

**Comment by nicoddemus** [comment]
Oops yeah my bad, missed that bit.

**Comment by nicoddemus** [comment]
Thanks for the heads up: https://github.com/pytest-dev/pytest/issues/7624

**Pull request merged**